libflagship/mqttapi.py

```diff
@@ -66,13 +71,16 @@ def on_message(self, client, userdata, msg, pkt, tail):
         pass
 
     @classmethod
-    def login(cls, printersn, username, password, key, ca_certs="ankermake-mqtt.crt", verify=True):
+    def login(cls, printersn, username, password, key, ca_certs=cert_path, verify=True):
         client = mqtt.Client()
 
         if verify:
             client.tls_set(ca_certs=ca_certs)
         else:
-            client.tls_set(ca_certs=ca_certs, cert_reqs=ssl.CERT_NONE)
+            log.warning('[Not Verifying Certificate]')
+            log.warning('This is insecure and should not be used in production environments.')
+            log.warning('It is recommended to run without "-k/--insecure".')
+            client.tls_set(ca_certs=None, cert_reqs=ssl.CERT_NONE)
```

Comment:
Let's keep libflagship simple, by defaulting `ca_certs=None`, and not having a real path default here. We are calculating it where needed anyway, so that'd be double the work.

This makes `libflagship` just a tool to be used, and keeps the glue logic in the cli part. Also, it's simpler, we just remove some lines :)

(must fix)